src/commands/admin/issue.ts

```diff
@@ -1,83 +1,65 @@
 /* eslint-disable camelcase */
-import { adminPerms } from '@lib/permissions';
+import { ADMIN_PERMS } from '@lib/permissions';
 import { RequestError } from '@octokit/types';
 import { BOT, GITHUB_PROJECT } from '@root/config';
 import { Command } from '@lib/types/Command';
-import { Message } from 'discord.js';
+import { ApplicationCommandOptionData, ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
 
 export default class extends Command {
 
-	usage = '<title>|[options]';
-	description = `Creates an issue in ${BOT.NAME}' repository.`;
-	extendedHelp = `You must pass in an issue title. Flags can be any of:
-	--labels=[comma, separated, list]\n--milestone=[milestone number]\n--project=[project (defaults to SageV2)]]`;
+	description = `Creates an issue in ${BOT.NAME}'s repository.`;
```

Comment:
You could specify comma separated here since the user sees the description when they go to use the slash command